src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
```

Comment:
this one seems to be redundant because all `stars__star` have bg of `images/star.svg`. the only reason you might need it is the previous selector.